pkg/porter/testdata/explain/expected-table-output.txt

```diff
@@ -0,0 +1,20 @@
+Name: HELLO
```

Comment:
🤩

Reply:
Is there supposed to be an emoji here? I see a square :/

Reply:
![Screen Shot 2019-09-26 at 2 13 01 PM](https://user-images.githubusercontent.com/1368985/65717795-c4d41f00-e067-11e9-8c10-c3714924d080.png)

GITHUB Y U NO HAVE STARSTUCK EMOJI!